test/unit/commands/gesture-specs.js

```diff
@@ -291,6 +291,28 @@ describe('gesture commands', () => {
       });
     });
 
+    describe('selectPickerWheelValue', () => {
+      const commandName = 'selectPickerWheelValue';
+
+      it('should throw an error if no mandatory parameter is specified', async () => {
+        await driver.execute(`mobile: ${commandName}`, {}).should.be.rejectedWith(/Element id is expected to be set/);
+        await driver.execute(`mobile: ${commandName}`, {element: 4}).should.be.rejectedWith(/is expected to be equal/);
+        await driver.execute(`mobile: ${commandName}`, {order: 'next'}).should.be.rejectedWith(/Element id is expected to be set/);
+      });
+
+      it('should throw an error if param is invalid', async () => {
+        await driver.execute(`mobile: ${commandName}`, {order: 'bla'}).should.be.rejectedWith(/is expected to be equal/);
```

Comment:
Since there is no element specified shouldn't this throw `Element id is expected to be set for selectPickerWheelValue method`?